package.json

```diff
@@ -41,9 +41,9 @@
 		"grunt": "^0.4.5",
 		"grunt-contrib-uglify": "^0.6.0",
 		"mocha": "^3.5.0",
-		"mocha-phantomjs": "^3.5.4",
+		"mocha-phantomjs": "^4.1.0",
 		"open": "0.0.5",
-		"phantomjs": "1.9.1 - 1.9.7-15",
+		"phantomjs-prebuilt": "2.1.16",
```

Comment:
Upstream changed the name to `phantomjs-prebuilt`, but otherwise it should be the same. 

The version pinning [was to accommodate `mocha-phantomjs`](https://github.com/mholt/PapaParse/pull/233#issuecomment-121745865), so it shouldn't be needed anymore.

Reply:
Ok about name change. 

I think we should use ^version to pin to all new compatible versions of prebuild package

Reply:
Whoops good catch. Fixed.